runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/fieldwatch/fw001.c

```diff
@@ -0,0 +1,265 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include <errno.h>
+#include "jvmti_test.h"
+#include <stdio.h>
+#include <string.h>
+
+jvmtiEnv *globalEnv;
+jfieldID currentFieldBeingWatched;
+const char * testClassName;
+jstring testClassString;
+
+
+    // Find the correct API to increment the number of reports we have seen and return the new value.
+    jint incAndGetNumReports(JNIEnv* jni_env)
```

Comment:
Why is the remainder of this file indented?

Reply:
I'm not familiar with the coding guidelines in this area of the codebase. I'll make the changes required. Would `openj9/runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/classModificationAgent/cma001.c` be a good example to look at as reference?

Reply:
@gacholio FYI ^

Reply:
Yes, that appears to be a good source (and even if it's not perfect, at the the test code will be internally consistent).

Also not you cannot use `//` comments in `.c` files.

Reply:
Fixed here: 51f520f